.github/workflows/check_cla.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     # Dont run this workflow if it was triggered by one of these bots
     # Don't run it during a merge queue, as it can't correctly identify the actor
-    if:  ${{ github.event.pull_request.user.login != 'dependabot[bot]' && github.event.pull_request.user.login != 'github-actions[bot]' && github.event.pull_request.user.login != 'sa-github-api' && github.event_name != 'merge_group' }}
+    if:  ${{ github.event.pull_request.user.login != 'dependabot[bot]' && github.event.pull_request.user.login != 'github-actions[bot]' && github.event.pull_request.user.login != 'sa-github-api' }}
```

Comment:
You probably want to also remove the comment above.
And the title of this PR is confusing.